gnovm/pkg/gnolang/preprocess.go

```diff
@@ -2837,6 +2978,13 @@
 				ct.String()))
 		}
 	case *FuncLitExpr:
+		for _, stmt := range cx.Body {
+			un = findUndefinedStmt(store, last, stmt, t)
```

Comment:
Nitpick: you can inline this:

```go
if findUndefinedStmt(store, last, stmt, t) != "" {
    return
}
```

Reply:
Yes but then if you want to debug, you need to introduce new variables. I am on the fence about this one.